components/Search.tsx

```diff
@@ -30,14 +37,20 @@ export default function Search({
   const [search, setSearch] = useState("");
   const [results, setResults] = useState<FormationData[]>([]);
   const [loading, setLoading] = useState(true);
+  const [tag, setTag] = useState("");
+  const allTags = cmsData.tags;
 
   const handleSearch = async (e: FormEvent) => {
     e.preventDefault();
 
     setLoading(true);
 
-    const query = encodeURIComponent(search);
-    const response = await fetch(`/api/search?q=${query}`);
+    const params = new URLSearchParams();
+    params.append("q", search);
+    if (tag !== "") {
+      params.append("tag", tag);
```

Comment:
Why not use just `t` here to follow the same convention as q for query? Doesn't really matter, just little details